### ISSUE TITLE
Automate the PR creation for plugins-json repository

### ISSUE THREAD

**Issue by abhi18av** [nextflow-io/nf-nomad#90]
We need to modify  the current release action so that upon release, the JSON file is used to eventually generate a PR for https://github.com/nextflow-io/plugins

**Comment by jagedn** [comment]
yes, agree, this is (ambitious) idea

**Comment by abhi18av** [comment]
<img src="https://media1.giphy.com/media/VspTn3CPKAHoA/giphy.gif"/>

**Comment by tomiles** [comment]
Yeah would be great, maybe it'd be a nice project for a hackathon, since this would be of wide interest outside of this project. Maybe something based on https://github.com/peter-evans/create-pull-request with some `jq` magic added.

**Comment by abhi18av** [comment]
Excellent idea @tomiles !  💯
